app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,26 @@
+class BookingsController < ApplicationController
+  before_action :set_event
+  before_action :set_booking, only: %i[cancel_booking]
+
+  def create
+    @booking = @event.bookings.new(params[:booking])
+    @booking.user = current_user
+
+    redirect_to root_path if @booking.save
+  end
+
+  def cancel_booking
+    @booking.update(canceled: true)
+    redirect_back fallback_location: root_path
+  end
```

Comment:
"booking" is implied by the fact we're in the BookingsController so it's a bit redundant to name this action "cancel_booking". Can you rename it "cancel" instead ?